schematools/importer/__init__.py

```diff
@@ -20,6 +20,7 @@
     "object": String,
     "boolean": Boolean,
     "integer": Integer,
+    "biginteger": BigInteger,
```

Comment:
```
schema validate 'https://schemas.data.amsterdam.nl/schema@v1.1.1#' tests/files/woningbouwplannen.json
Structural validation: 
'biginteger' is not one of ['array', 'boolean', 'integer', 'null', 'number', 'object', 'string']

Failed validating 'enum' in schema[0]['properties']['type']:
    {'enum': ['array',
              'boolean',
              'integer',
              'null',
              'number',
              'object',
              'string']}

On instance['type']:
    'biginteger'
Reading schema from tests/files/woningbouwplannen.json
Semantic validation: success!
```

Reply:
I think we both misread the spec https://github.com/Amsterdam/amsterdam-schema/wiki/Amsterdam-Schema-Specification . Well at least I did.

The only valid JSON schema integral types are `integer` and `number`. The types mentioned in brackets after `integer`, eg:
```
integer (tiny8, small16, int32, bigint64)
```
are the types they map to depending on the values of minimum and maximum as you originally suggested. I'm afraid you did all this work due to me misreading it. I'm sorry

Reply:
What I'm saying is that according to JSON schema we can't have `biginteger` as a type on its own.

Reply:
Only realized that when I ran the validator and saw the error message.

Reply:
See https://github.com/Amsterdam/amsterdam-schema/wiki/Grootst-Gemene-Deler  and search for `exclusiveMaximum` to see how integers were meant to be expressed.